src/plugin/leaflet.canvas-markers.js

```diff
@@ -178,12 +191,12 @@ function layerFactory(L) {
             var iconSize = marker.options.icon.options.iconSize;
 
             var adj_x = iconSize[0]/2;
-            var adj_y = iconSize[1]/2;
+            var adj_y = iconSize[1];
```

Comment:
Instead of taking the full size, the ret below should be using the X/Y Anchor for the adjustment. 
I'm assuming your image has an anchor adjustment which is not being accounted for in the hover-over\click box of the icon, so the mouse hover over interactivity is off from what you are seeing drawn.

Reply:
Yes there was something weird there. Here is the marker code I had:

I didn't use anything weird in the code, so seems like this library should be able to automatically adjust shouldn't it?

Removing the /2 %100 fixed it.

    var customIcon = L.icon({
        iconUrl: myicon,
        iconSize: [35, 35], // size of the icon
        iconAnchor: [17.5, 35],
        popupAnchor: [0, -40]
    });

Reply:
Its not exactly something being weird, just using it in a manor which myself and others aren't. I really don't use anchors often if at all, so it is definitely something I would notice during my day to day use.

Thanks for the code though, i'll try to play around with it and adjust things.